analytic_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Analytic Operating Unit",
+    "version": "12.0.1.0.0",
+    "author": "Eficent, "
+              "ERP Harbor Consulting Services, "
```

Comment:
@sudhir-erpharbor I think to remove `author`.

@aheficent Can you help here?

Reply:
Thanks @nikul-serpentcs for your suggestion.
AFAIK, If we are migrating or contributing to the module, we can add us in the author.
@jbeficent  @aheficent Would guys please share some lights on it whether to keep the author or remove it?
Thanks,

Reply:
IMHO, I think the authors key is reserved for:
- The original authors
- Contributors that make significant changes in the behavior of the module (add new features, adapt to dramatic changes in Odoo, etc)

For standard migration I think that adding the person to the contributors section is enough. For example @lreficent and @gdgellatly contributed to migrate and they are not in the authors section.

Despite all that I 'm not the author of this, it is just my opinion.

Regards.

Reply:
@sudhir-erpharbor @aheficent Thank You for Support !!